src/index.js

```diff
@@ -25,7 +25,10 @@ async function main() {
     projectRoot,
     './videos',
     getFormattedDate(),
-    `output_${configuration.size}_${configuration.period}_${configuration.language}`,
+    `${Object.entries(configuration)
+      .map(([key, value]) => value) // eslint-disable-line no-unused-vars
```

Comment:
You could use `.map(([, value]) => value)` so you don't have to disable `no-unused-vars`, but you should use `Object.values(configuration)` instead of `Object.entries(configuration)` since you don't need the keys.